scripts/cluster/setup_master_node.go

```diff
@@ -199,7 +214,13 @@ func InstallKnativeServingComponent(stockContainerd string) error {
 		if !utils.CheckErrorWithMsg(err, "Failed to install Knative Serving component!\n") {
 			return err
 		}
-		_, err = utils.ExecShellCmd("kubectl apply -f https://github.com/knative/serving/releases/download/knative-v%s/serving-core.yaml", configs.Knative.KnativeVersion)
+		
+		loaderServingCorePath, err := utils.GetVHiveFilePath(path.Join("configs/knative_yamls", "loader_serving_core.yaml"))
```

Comment:
Please rename these files to be clearer what each one does: we have one for containerd (with stock version of knative images) and for firecracker (with custom versions).

In the end, there should be no `loader` in the names of these files.